src/main/java/org/ow2/proactive/catalog/Application.java

```diff
@@ -91,6 +91,7 @@
     private String dataSourcePassword;
 
     public static void main(String[] args) {
```

Comment:
Why do we even have main method?

Reply:
It is the entry point for spring boot to start the service